cms/lib/xblock/tagging/tagging.py

```diff
@@ -43,7 +43,8 @@ def student_view_aside(self, block, context):  # pylint: disable=unused-argument
         Display the tag selector with specific categories and allowed values,
         depending on the context.
         """
-        if isinstance(block, CapaModule):
```

Comment:
Hmm, I didn't know about ``is_author_mode``. It looks like @jimabramson added this to support a special discussion rendering for authors:

https://github.com/edx/edx-platform/pull/3295

A couple of months later I added author_view as a similar solution to allow an XBlock to render a different experience for authors in Studio:

https://github.com/edx/edx-platform/pull/3999

We should really refactor the former to use ``author_view`` but that can be for another PR. For your issue, I think we need to work out why asides aren't rendering for author views.